go.sum

```diff
@@ -3,6 +3,8 @@ github.com/Azure/azure-sdk-for-go v10.3.0-beta+incompatible/go.mod h1:9XXNKU+eRn
 github.com/Azure/go-autorest v9.10.0+incompatible/go.mod h1:r+4oMnoxhatjLLJ6zxSWATqVooLgysK6ZNox3g/xq24=
 github.com/Azure/go-ntlmssp v0.0.0-20170803034930-c92175d54006/go.mod h1:chxPXzSsl7ZWRAuOIE23GDNzjWuZquvFlgA8xmpunjU=
 github.com/ChrisTrenkamp/goxpath v0.0.0-20170625215350-4fe035839290/go.mod h1:nuWgzSkT5PnyOd+272uUmV0dnAnAn42Mk7PiQC5VzN4=
+github.com/Didainius/go-vcloud-director/v2 v2.2.0-alpha.1.0.20190417085158-6cbca97d9053 h1:xzzeqQY7gih0ZxS8h0ubOwmeosv+XsCtdbiHDlovLHc=
+github.com/Didainius/go-vcloud-director/v2 v2.2.0-alpha.1.0.20190417085158-6cbca97d9053/go.mod h1:HonlGxbjJ1NAibWh99eE4/S2l6ZOZ5KJzKK1rh2a9vc=
```

Comment:
I believe we shouldn't be putting personal repo references to master. Better to prepare a new tag in govcd.

Reply:
This is a consequence of using the "replace" method. It will go away after we remove the replacement